server/lib/poller.js

```diff
@@ -65,29 +65,61 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
```

Comment:
A `let` variable has a scope lifetime; unfortunately babel translates them to `var`, which hoists them to the top of the current function. But with real ES6, `checkAccounts` would be defined only in the `try` block. Can you declare it outside the try block, please?